docs/os/security.md

```diff
@@ -0,0 +1,42 @@
+
+At first, only a single public API endpoint for pairing is available that allows the two components to establish an encrypted communication channel. To rule out a man-in-the-middle attack, a confirmation code is shown both within the App and on the Base and needs to be confirmed manually. With the secure connection established, the [Noise encryption](http://noiseprotocol.org) can be set up and the full API functionality is available to the App.
+
+### Networking
+The following components help harden the Base against networking attacks:
+
+* **No web interface**: Modern web applications, built conveniently with frameworks like Node.js or Flask, have hundreds of dependencies that are outside our control and could contain security vulnerabilities or even malicious code. This is why we decided not to provide a web interface directly on the Base and expose only a single API by default.
+
+* **NGINX**: although the Middleware exposes only a single API, additional components like electrs can optionally provide their own public API, e.g. for direct usage with the Electrum wallet. To have only a single entry-point from the public network into the Base, all communication is routed through [NGINX](https://www.nginx.com), which is a enterprise-grade, reliable and secure reverse proxy server.
+
+* **Firewall**: to mitigate against network snooping and the exploitation of potentially open ports on operating system level, very restrictive packet filtering rules are set with [iptables](https://netfilter.org/projects/iptables/index.html) to refuse any connections that are not explicitely allowed.
+
+* **Brute-force protection**: by default, SSH-login is disabled. Once enabled, only login with SSH keys should be used. Nonetheless, we use [fail2ban](https://www.fail2ban.org) to log any login attempts and block ip addresses for a certain time after too many unsuccessful tries.
+
+* **SSH Keys**: for secure usage of SSH, the password login is disabled by default, so that a terminal login is only possible using SSH keys. The public key can be integrated in the Base image on build time, or later with the App.
+
+* **Disabled root user**: SSH login for the root user is disabled by default, even using SSH keys. This way, even when gaining access to the device using a stolen key, the attacker still needs to enter the root password when using `sudo`.
+
+* **Service users**: every service runs as a dedicated service user, with as little privileges as possible.
+
+### Local security
+A network appliance like the BitBox Base, built on readily available components, can never be assumed to be safe against an attacker that has physical access to the device. Nonetheless, device should be safe against casual tampering.
+
+* **Tamper-evident opening**: by providing support for tamper-evident seals, the device can be secured in a way that makes it evident if it has been opened.
```

Comment:
For my information, how do we "provid[e] support for tamper-evident seals"?

Reply:
I can take that out for the moment, it's more of an idea still.